src/models/contracts/objectExplorer/nodeInfo.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ObjectMetadata } from 'vscode-mssql';
+import * as vscodeMssql from 'vscode-mssql';
```

Comment:
nit: `vscodeMssql` isn't the easiest on the eyes when reading code. 😄 Maybe just `mssql` would "look" better.